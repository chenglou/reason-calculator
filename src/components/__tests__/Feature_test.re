open Jest;
open Expect;

let _ =

describe "Feature" (fun _ => {
  test "renders checkmark emoji" (fun _ => {
    let emoji = Feature.Feature.Checkmark;
    let text = "A feature";
    let tree = Feature.createElement ::emoji ::text children::[] ()
      |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot
  });

  test "renders soon emoji" (fun _ => {
    let emoji = Feature.Feature.Soon;
    let text = "A feature";
    let tree = Feature.createElement ::emoji ::text children::[] ()
      |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });
});
