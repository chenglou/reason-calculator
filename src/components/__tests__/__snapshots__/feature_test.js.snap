// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature renders checkmark emoji 1`] = `
<div
  className="css-pat51 css-vlxvfg"
>
  <Checkmark
    reasonProps={0}
  />
  <h3>
    A feature
  </h3>
</div>
`;

exports[`Feature renders soon emoji 1`] = `
<div
  className="css-pat51 css-vlxvfg"
>
  <Soon
    reasonProps={0}
  />
  <h3>
    A feature
  </h3>
</div>
`;
